src/components/saveAccount/saveAccount.js

```diff
@@ -1,36 +1,47 @@
 import React from 'react';
 import InfoParagraph from '../infoParagraph';
 import ActionBar from '../actionBar';
-import { setSavedAccount } from '../../utils/saveAccount';
+import networksRaw from '../login/networks';
 
-export default class SaveAccount extends React.Component {
-  save() {
-    setSavedAccount(this.props.account);
-    this.props.closeDialog();
-    this.props.successToast({ label: 'Account saved' });
-    this.props.done();
-  }
+const SaveAccount = ({
+  network,
+  address,
+  publicKey,
+  closeDialog,
+  accountSaved,
+}) => {
+  const save = () => {
+    // eslint-disable-next-line arrow-body-style
+    const index = networksRaw.map((item, i) => {
+      return (item.name === network) ? i : null;
+    }).filter(item => item !== null)[0];
+    accountSaved({
```

Comment:
`.find(item => item ! == null);` will return first element that not equal null

Reply:
Done